scripts/src/main/resources/scripts/command/java

```diff
@@ -96,10 +92,197 @@ function doCreate() {
       cd "${DEVON_IDE_HOME}/workspaces/main" || exit 255
     fi
   fi
-  doMavenArchetype "${@}"
+
+  local framework
+  local mavenargs
+  local package
+  local artifactId
+  while [[ -n "${1}" ]]
+  do
+    if [[ "${1}" == "-f" ]] || [[ "${1}" == "-framework" ]]  
+    then
+      framework=${2}
+      shift
+    elif [[ "${1:0:8}" == "-Ddbtype" ]]
+    then 
+      dbtype=${1:9}
+    elif [[ "${1:0:2}" == "-D" ]]
+    then
+      mavenargs="${mavenargs} ${1}"
+    elif [[ "${1}" =~ [a-z]+[.][a-z]+([.][a-z]+)* ]]
+    then 
+      package="${package} ${1}"
+    elif [[ "${1}" =~ [^.]* ]]
+    then
+      ## some better definition for a artifactID TODO
+      artifactId+=${1}
+    fi
+    shift
+  done
+
+  while [[ -z "${framework}" ]] || [[ "${framework}" != "spring" ]] && [[ "${framework}" != "quarkus" ]]
+  do
+    if [[ -n ${package} ]]
+    then
+      framework="spring"
+      break
+    fi
+    if [[ -z "${framework}" ]]
+    then
+      doEcho "No framework selected." 
+    fi
+    read -r -p "(Select a framework: 'quarkus' or 'spring'): " framework
+  done
+
+  if [[ "${framework}" = "quarkus" ]]
+  then
+    startQuarkusProject "${dbtype}"
+  elif [[ "${framework}" = "spring" ]]
+  then
+    if [[ -z "${package}" ]] && [[ -z "${mavenargs}"  ]]
+    then 
+      doEcho "No package or other parameter passed, we need at least a package to create a spring application more information https://devonfw.com/website/pages/docs/devonfw-guide_ide.wiki_cli.asciidoc.html#devonfw-guide_ide.wiki_java.asciidoc_create" 
+      read -r -p "Pass a package and optional parameters as shown in the examples: " mavenargs
+    fi
+    doMavenArchetype "${package}" "${mavenargs}" "${dbtype}" "${artifactId}"
+  fi
   exit ${?}
 }
 
+# dbtype
+function startQuarkusProject() {
+  local message='\nPlease select your preferred extensions and set the project group-, artifactId-, and version
+  on the opened site https://code.quarkus.io/ or just press return to use a configured project from our 
```

Comment:
how about doing this additionally:
```
doOpen "https://code.quarkus.io/"
```

This will open your default browser (or a new tab in it) and go to that URL.
Therefore the user does not have to copy&paste the URL manually.